ddp-workspace/projects/ddp-testboston/src/app/app.module.ts

```diff
@@ -97,6 +98,8 @@ sdkConfig.doLocalRegistration = DDP_ENV.doLocalRegistration;
 sdkConfig.mapsApiKey = DDP_ENV.mapsApiKey;
 sdkConfig.auth0Audience = DDP_ENV.auth0Audience;
 sdkConfig.projectGAToken = DDP_ENV.projectGAToken;
+sdkConfig.errorReportingApiKey = DDP_ENV.errorReportingApiKey;
```

Comment:
I've accidentally pushed errorReportingApiKey to the github repo, the key was re-generated.

Reply:
Not a big deal--this will show up in the frontend deployment anyway, and we will need to lock it down by origin in the GCP console anyway.  We'll need to generate 4 env specific keys for this...